ext/reflection/php_reflection_arginfo.h

```diff
@@ -846,21 +857,25 @@ ZEND_METHOD(ReflectionFiber, getExecutingLine);
 ZEND_METHOD(ReflectionFiber, getCallable);
 ZEND_METHOD(ReflectionFiber, getTrace);
 
+
 static const zend_function_entry class_ReflectionException_methods[] = {
 	ZEND_FE_END
 };
 
+
 static const zend_function_entry class_Reflection_methods[] = {
 	ZEND_ME(Reflection, getModifierNames, arginfo_class_Reflection_getModifierNames, ZEND_ACC_PUBLIC|ZEND_ACC_STATIC)
 	ZEND_FE_END
 };
 
+
 static const zend_function_entry class_Reflector_methods[] = {
 	ZEND_FE_END
 };
 
+
 static const zend_function_entry class_ReflectionFunctionAbstract_methods[] = {
-	ZEND_RAW_FENTRY("__clone", zim_ReflectionClass___clone, arginfo_class_ReflectionFunctionAbstract___clone, ZEND_ACC_PRIVATE, NULL, NULL)
+	ZEND_MALIAS(ReflectionClass, __clone, __clone, arginfo_class_ReflectionFunctionAbstract___clone, ZEND_ACC_PRIVATE)
```

Comment:
I may have use the wrong method to regenerate this file, any hint to make this correct ?

Reply:
You generally don't need to do anything, the makefile regenerates this file automatically when you modify the stubs. But the full command is `php build/gen_stub.php -f --generate-optimizer-info`.

Reply:
Thanks, i wrongly assume that i had to do a `make php_reflection_arginfo.h` in this directory, should be fixed then